plugins/manipulator.cpp

```diff
@@ -198,7 +198,7 @@ const SkillColumn columns[] = {
     {12, 4, profession::NONE, unit_labor::PULL_LEVER, job_skill::NONE, "Lv"},
     {12, 4, profession::NONE, unit_labor::BUILD_ROAD, job_skill::NONE, "Ro"},
     {12, 4, profession::NONE, unit_labor::BUILD_CONSTRUCTION, job_skill::NONE, "Co"},
-    {12, 4, profession::NONE, unit_labor::REMOVE_CONSTRUCTION, job_skill::NONE, "CR"},
+    {12, 4, profession::NONE, unit_labor::UNUSED_13, job_skill::NONE, "CR"},
```

Comment:
This line should just go away entirely

Reply:
I was worried that it would misalign something else, so I left it in (for now, at least)